### PR TITLE
Update st2 command aliases

### PR DESCRIPTION
**Pull request by Kami**

### CODE REVIEW

File: packs/st2/aliases/actions_list.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ---
-name: "st2-actions-list"
+name: "st2_actions_list"
 action_ref: "st2.actions.list"
 description: "List available StackStorm actions."
+formats:
+    - "list actions"
```
Comment (Kami) [inline packs/st2/aliases/actions_list.yaml]: I think we should probably still prefix all those commands with "st2" (e.g. `!st2 list actions`). 

What do other ppl think?

Comment (jfryman) [inline packs/st2/aliases/actions_list.yaml]: I do prefer that.

Comment (manasdk) [inline packs/st2/aliases/actions_list.yaml]: Yes prefix `st2`

**Comment by manasdk** [comment]
+1

**Pull request merged**